packages/postgres/src/loader.ts

```diff
@@ -23,7 +23,10 @@ export function loadSql(filename: string): SqlFile {
   return sql;
 }
 
-export function loadSqlEquiv(filePathOrUrl: string): SqlFile {
+export function loadSqlEquiv(filePathOrUrl: string | undefined): SqlFile {
+  if (!filePathOrUrl) {
+    return {};
+  }
```

Comment:
What's going on here? Calling this with `undefined` will be an error 100% of the time.